tools/static-assets/skel-full/imports/api/links/links.js

```diff
@@ -2,4 +2,6 @@
 
 import { Mongo } from 'meteor/mongo';
 
-export const Links = new Mongo.Collection('links');
+const Links = new Mongo.Collection('links');
+
+export { Links };
```

Comment:
Writing the exported variable twice? It just seems unnecessary. If there are no advantages then the most concise format wins IMO